gifted_build/src/components/second_page.tsx

```diff
@@ -1,10 +1,37 @@
 import * as React from "react";
+import "../css/second_page.css"
 
 class Second_Page extends React.Component<{}, {}> {
     render () {
         return (
-            <div style={{height: '800px'}}>
-                TBD - Under Construction
+            <div id="container">
+                <div id="our_partners_container">
+                    <div id="our_partners_text_container">
+                        <h1 className="our_partners_title">OUR PARTNERS </h1>
+                        <p className="summary_text"> Here at Gifted Clothing Brand, LLC, we take pride in our 
+                            stance on self-awareness and philanthropy. Because of this, 
+                            we strive to continuously expand our Gifted family by 
+                            partnering with non-profit organizations. Thank you for 
+                            your continued support to us and our partners!</p>
+                    </div>
+                    <div id="handshake_img">
+                        <img src={require('../files/Handshake_edited.jpeg')} />
+                    </div>
+                </div>
+                <div id="spolympics_container">
```

Comment:
like wise with the comment in the special olympics page, this shouldn't be hardcoded and should be variable with the data pulled from behind the database. Right now this is fine as we do not have the back end set up, but this should be named "spolympics" rather something like "partner_entry" to allow changes to the partner page

Reply:
Will work on this after it's merged as well!